### PR TITLE
master miner core changes

### PR DESCRIPTION
**Pull request by shaishav11**
Created a new miner called Master.Risklist Miner for Recorded Future Inc as discussed with Mr. Luigi Mori to fetch data with following features: 
1.) by customizing risklist path (Fusion) 
2.) by specifying a specfic risklist (Connect API)

### CODE REVIEW

File: minemeld/ft/recordedfuture.py [hunk 1]
```diff
@@ -22,11 +22,10 @@
 import netaddr
 import netaddr.core
 
-from . import csv
+from minemeld.ft import csv                      # changed from . to minemeld.ft
```
Comment (jtschichold) [inline minemeld/ft/recordedfuture.py]: Why?

File: nodes.json [hunk 2]
```diff
@@ -130,8 +133,5 @@
     },
     "minemeld.ft.visa.VTI": {
         "class": "minemeld.ft.visa:VTI"
-    },
-    "minemeld.ft.taxii2.Taxii2Client": {
```
Comment (jtschichold) [inline nodes.json]: Why Taxii2Client was removed?

**Comment by jtschichold** [comment]
Thank you for the PR! PRs should target `develop` not `master`. Please close this PR and open a new one targeting `develop`

**Comment by sshah1103** [comment]
Thank you for guiding. I have raised a new pull request with: New Master Miner code modifications #353 targeting --> develop

I have also commented all the changes and added Taxii2Client. It was unintentionally deleted while experimenting.

Closing this request now.

**Pull request closed without merging**